Tests/ViewControllers/BrickCollectionViewTests.swift

```diff
@@ -738,7 +728,7 @@ class BrickCollectionViewTests: XCTestCase {
 
     func testClassNibIdentifiers() {
         let brick = LabelBrick("Label", text: "Hello")
-        brickView.registerBrickClass(LabelBrick.self, nib: LabelBrickNibs.Image)
+        brickView.registerNib(LabelBrickNibs.Image, forBrickWithIdentifier: "Label")
```

Comment:
registerNib should be a separate issue. That would require a refactor of a lot of the example app and tests.